### DISCUSSION TITLE
MouseInc这个用360浏览器，关闭窗口手势一下子关闭全部了，Google Chrome Portable可以关闭单个

### DISCUSSION THREAD

**Discussion by pengmingz** [shuax/MouseInc.Settings#293] [Q&A] [answered]
MouseInc这个用360浏览器，关闭窗口手势一下子关闭全部了，Google Chrome Portable可以关闭单个窗口，求大神解决谢谢
测试很多次都这样，浏览器更新最新版也不行，重启电脑都不能解决，求大佬看看咋回事，用360自带手势就正常，可以关闭单独标签，但是用Mouselnc就不行，一下子就关闭全部了，很是尴尬，我习惯手势关闭浏览器，这一整太不方便了

**Comment by gbsix** [comment]
排除360浏览器就行了。

**Comment by Ctrl-Ling** [comment]
那你改成用关闭页签的手势嘛，注入按键Ctrl+W关闭页签，关闭窗口就是关闭窗口，关闭页签就是关闭页签，各司其职逻辑统一才是最好的，相反你关闭窗口的手势只触发关闭页签才是不正常的

**Comment by shuax** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/6940583/175505150-ec32c2ea-b347-4adb-846c-0165dd005cb5.png)

自己把程序名加进去。

**Reply by gbsix** [reply]
你不说我都不知道，用了这么久，怎么就没发现这个功能呢！ 一定是我对“ 特定动作”这几个字理解有误。这下clover一下子把全部标签都关掉的问题也解决了！

**Reply by pengmingz** [reply]
怎么添加单独关闭页标签呢，看着设置就是关闭页标签啊，但是时间操作 是执行 关闭所有标签
之前不知这样，突然这样了，

**Reply by pengmingz** [reply]
#
![image](https://user-images.githubusercontent.com/107525130/175517090-20135446-e4a3-4851-8ab2-ddc62283b041.png)

**Reply by pengmingz** [reply]
好像版本不一样

**Reply by gbsix** [reply]
点击 程序列表

**Reply by pengmingz** [reply]
没有那个选项，没找到程序列表

**Reply by gbsix** [reply]
看你的截图，“有效”两个字上面，默认折叠了。

**Reply by WTTW113** [reply]
你添加360进去之后就可以增添手势
![image](https://user-images.githubusercontent.com/110770874/183290612-fd98b98e-56e2-4fde-bf36-c3ec1a72f883.png)
